dev/run

```diff
@@ -249,9 +249,15 @@ def hack_default_ini(ctx, node, content):
     coffeejs = os.path.join(ctx['rootdir'], "share", "server", "main-coffee.js")
 
     repl = "javascript = %s %s" % (couchjs, mainjs)
+    if os.name == 'nt':
+        # Erlang always wants UNIX-style paths
+        repl = repl.replace("\\", "/")
```

Comment:
I'm sorry for the confusion. I was not clear enough. When I typed `+1` on IRC I meant that I gave a +1 to this PR as is.
If you believe that the approach we discussed is better. We would have to convert `string.replace` into new approach. However in this case you would modify lot's of places anyway so I would suggest to create a `normalize_path` function in the module which would have `if os.name == 'nt'` logic inside.